R/descriptives.R

```diff
@@ -1274,8 +1274,9 @@ Descriptives <- function(jaspResults, dataset, options) {
     tb <- as.data.frame(table(x))
     scaleX <- ggplot2::scale_x_discrete(limits = factor(tb[,1]))
```

Comment:
, 1

Reply:
I'd prefer not to adjust too much code before the release, however minor it may be, so please keep it limited to the function I changed. There are many other stylistic issues in descriptives (e.g., whitespace, snake_case) which I'd prefer to leave for after the release.